processes/oclcClassify.py

```diff
@@ -122,23 +122,24 @@ def frbrizeRecord(self, record):
     def classify_record_by_metadata_v2(self, identifier, identifier_type, author, title):
         search_query = self.oclc_catalog_manager.generate_search_query(identifier, identifier_type, title, author)
 
-        # TODO: SFR-2090: Finalize query to brief bibs
-        related_oclc_bibs = self.oclc_catalog_manager.query_brief_bibs(search_query)
+        related_oclc_bibs = self.oclc_catalog_manager.query_bibs(search_query)
```

Comment:
query_bibs would recover gracefully from a 400, but I think we could just return the v2 classify function or otherwise short circuit the call to OCLC with a reasonable error if there's no query (as long as we trust the original logic, that it's not worth trying to query on title alone, e.g.).